docs/introduction/friendly-url-breadcrumb-navigation.md

```diff
@@ -34,3 +34,18 @@ The rest of the work is done automatically and URLs provided by these providers
 
 *Note: You can use [`UrlListType` in your forms](https://github.com/shopsys/shopsys/blob/master/docs/introduction/using-form-types.md#urllisttype) to edit friendly URLs of existing entities.*
 *If you're interested, you can take a look at the processing of `Article` entity (see `ArticleFacade`, `ArticleData::$urls`, `ArticleDataFactory` and `ArticleFormType`), which allows for this functionality.*
+
+# Breadcrumb navigation
```

Comment:
It would make sense to me to have two different articles. They really don't have that much in common, it looks like two articles (albeit similar) appended together.

I would be fine with having them together in the FAQ:
```markdown
## How can I create a friendly URL or Breadcrumb navigation for my entity?
See the articles [Friendly URL](/docs/introduction/friendly-url.md) and [Breadcrumb Navigation](/docs/introduction/breadcrumb-navigation.md).
```

Also, according to [Guidelines for Writing Documentation](https://github.com/shopsys/shopsys/blob/master/docs/contributing/guidelines-for-writing-documentation.md) the  secord word of the heading should be capitalized: *Breadcrumb Navigation*.

Reply:
Well I am against that till we have somehow better navigation in documentation. Till then I really think it is better to group articles with similar content.

Reply:
I have updated capitalization of words

Reply:
These are just two articles appended to each other. Breadcrumb navigation is required for every new front-end page, friendly URLs are less used. This merging of different articles imo makes the docs less usable.

Imagine you're trying to see how to create breadcrumb navigation. You go to FAQ and find [the answer](https://github.com/shopsys/shopsys/blob/f0eef8bf192dde6d0d50b90ebe831f08b8473bdb/docs/introduction/faq-and-common-issues.md#how-can-i-create-a-friendly-url-or-front-end-breadcrumb-navigation-for-my-entity). You click the link. Now it seems that you are in a wrong place, unless it occurs to you to scroll. This will become a bigger issue when the Friendly URL part gets longer.

If it was true that every time you create a Friendly URL you'll need a Front-end Breadcrumb Navigation altered and vice versa, it would make sense to glue them together. In that case, there should be a proper heading of such article *"Friendly URL, Front-end Breadcrumb Navigation"* and some introduction that explains that these concepts are interlinked.
In my opinion, this is not the case. For every front-end page, you'll need Breadcrumb Navigation, but only for a subset you'll need Friendly URLs. So in my view, it would make sense to link those articles in a note (*"Note: You're creating a new front-end route - don't forget to configure Breadcrumb Navigation for the new page."* / *"Note: For some front-end pages you might also like to set up Friendly URLs."*).

Reply:
If I had to vote, I would go for separation :wink:

Reply:
Two against one - I have separated it